src/spikeinterface/postprocessing/correlograms.py

```diff
@@ -16,35 +16,53 @@
 
 class ComputeCorrelograms(AnalyzerExtension):
     """
-    Compute auto and cross correlograms.
+    Compute auto and cross correlograms of unit spike times.
 
     Parameters
     ----------
     sorting_analyzer: SortingAnalyzer
         A SortingAnalyzer object
     window_ms : float, default: 50.0
-        The window in ms
+        The window around the spike to compute the correlation in ms. For example,
+         if 50 ms, the correlations will be computed at lags -25 ms ... 25 ms.
     bin_ms : float, default: 1.0
-        The bin size in ms
+        The bin size in ms. This determines the bin size over which to
+        combine lags. For example, with a window size of -25 ms to 25 ms, and
+        bin size 1 ms, the correlation will be binned as -25 ms, -24 ms, ...
     method : "auto" | "numpy" | "numba", default: "auto"
-         If "auto" and numba is installed, numba is used, otherwise numpy is used
+         If "auto" and numba is installed, numba is used, otherwise numpy is used.
 
     Returns
     -------
-    ccgs : np.array
+    correlogram : np.array
         Correlograms with shape (num_units, num_units, num_bins)
-        The diagonal of ccgs is the auto correlogram.
-        ccgs[A, B, :] is the symetrie of ccgs[B, A, :]
-        ccgs[A, B, :] have to be read as the histogram of spiketimesA - spiketimesB
+        The diagonal of correlogram is the auto correlogram. The output
+        is in bin counts.
+        correlogram[A, B, :] is the symmetry of correlogram[B, A, :]
```

Comment:
I don't understand what "the symmetry of" means. Just that `correlogram[A, B, :] = correlogram[B, A, :]`?

Reply:
`correlogram[A, B, :] = correlogram[B, A, ::-1]`

Reply:
I think further to the discussion below on strange bin-edge behaviour, it is easier to not make an explicit statement on the nature of the bin symmetry. How about:

```
The diagonal of the correlogram (e.g. correlogram[A, A, :]) holds the unit auto correlograms. The off-diagonal elements are the cross-correlograms between units, where correlogram[A, B, :] and correlogram[B, A, :] represent cross-correlation between the same pair of units, applied in opposite directions.
```

Reply:
I think this is still a bit ambiguous, you could read it quickly and think "Oh yes, A, B is in the opposite order as B, A, that makes sense." So I'd also vote for the equation. You could just add `so that correlogram[A, B, :] = correlogram[B, A, ::-1]` at the end of that description?

Reply:
Agree thanks both have added, initially I thought maybe the formula was not true in all edge cases, but looks like it is!